File: tic_tac_php/player.php
<?php

class Player {
    private string $name;
    private string $mark;

    public function __construct($name, $mark) {
        $this->name = $name;
        $this->mark = $mark;
    }

    public function __get($name) {
        return $this->$name;
    }
}
?>

File: tic_tac_cs/Player.cs
namespace tic_tac_cs
{
    internal class Player
    {
        private string name;
        private string mark;

        public Player(string name, string mark)
        {
            this.name = name;
            this.mark = mark;
        }

        public string Name
        {
            get { return name; }
        }

        public string Mark
        {
            get { return mark; }
        }
    }
}
